src/config/theme/style.rs

```diff
@@ -60,6 +60,13 @@ impl RawAppStyle {
             "light_cyan" => style::Color::LightCyan,
             "white" => style::Color::White,
             "reset" => style::Color::Reset,
+            s if s.starts_with('#') => {
+                let rgb = Rgb::from_hex_str(s).unwrap();
```

Comment:
Rather than unwrap, we should handle the error and return `style::Color::Reset`

Reply:
i tried that with a `match` statement like you did, but it turns out that when you do that you get errors because it gets a different result on error

Reply:
i'm not exactly sure What error to parse, is all

Reply:
Try something like and see if that works
```rust
s if s.starts_with('#') => {
    let rgb = match Rgb::from_hex_str(s) {
        Ok(s) => s,
        _ => return style::Color::Reset,
    };
    let r = rgb.get_red() as u8;
    let g = rgb.get_green() as u8;
    let b = rgb.get_blue() as u8;
    style::Color::Rgb(r, g, b)
},
```

Reply:
yeah that works. will push this

Reply:
yeah that works. will push